themes/mlibrary/custom.php

```diff
@@ -273,23 +290,24 @@ function mlibrary_exhibit_builder_attachment($html, $compact) {
   //$imageSize = $compact['fileOptions']['imageSize'];
 
   // All Exhibit builder layout out of the box. Only customization for those layouts is adding video item type
```

Comment:
These lines look like whitespace-only changes. I'd encourage leaving those out of commits if you can help it.